openedx/core/djangoapps/discussions/tests/test_views.py

```diff
@@ -0,0 +1,101 @@
+"""
+Test app view logic
+"""
+# pylint: disable=test-inherits-tests
+from django.urls import reverse
+from opaque_keys.edx.keys import CourseKey
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from common.djangoapps.student.tests.factories import UserFactory
+from lms.djangoapps.courseware.tests.factories import StaffFactory
+from lms.djangoapps.courseware.tests.factories import GlobalStaffFactory
```

Comment:
TODO: Move these helpers to a shared location, presumably `common.djangoapps.student.tests.factories`?

Nothing there seems LMS-specific and could/should be shared across the platform; they're useful :)

Reply:
See: https://github.com/edx/edx-platform/pull/27329